accounts/abi/bind/precompile_config_template.go

```diff
@@ -14,14 +14,14 @@ const tmplSourcePrecompileConfigGo = `
 // For testing take a look at other precompile tests in core/stateful_precompile_test.go and config_test.go in other precompile folders.
 
 /* General guidelines for precompile development:
-1- Read the comment and set a suitable contract address in precompile/params.go. E.g:
-	{{.Contract.Type}}Address = common.HexToAddress("ASUITABLEHEXADDRESS")
-2- Set gas costs in contract.go
+1- Read the comment and set a suitable contract address in generated contract.go. E.g:
+	ContractAddress = common.HexToAddress("ASUITABLEHEXADDRESS")
+2- Set gas costs in generated contract.go
 3- It is recommended to only modify code in the highlighted areas marked with "CUSTOM CODE STARTS HERE". Modifying code outside of these areas should be done with caution and with a deep understanding of how these changes may impact the EVM.
```

Comment:
One thing might be helpful is to move set gas costs to one of the later steps, or right after step 3. Most people don't know how many reads or writes they'll do until after they implement their precompile function.

Reply:
done